test/mainnet-fork/ForkTestBase.t.sol

```diff
@@ -41,13 +41,15 @@ interface IBufferLike {
 }
 
 interface IPSMLike {
+    function bud(address) external view returns (uint256);
     function pocket() external view returns (address);
     function kiss(address) external;
     function rush() external view returns (uint256);
 }
 
 interface IVaultLike {
     function rely(address) external;
+    function wards(address) external returns (uint256);
 }
 
 contract ForkTestBase is DssTest {
```

Comment:
To ensure consistency between integration tests and the spell, I would change the `setup` here to use the deploy/init libraries instead of manually setting it all up.

Reply:
I'm gonna make a ticket for this and do in a separate PR cause this is already quite big.

Reply:
https://linear.app/phoenixlabs/issue/SC-704/refactor-test-suite-to-use-deployed-mainnet-contracts-and-deployinit

This is good cause I wanted to refactor as well to use deployed USDS etc so can do that as part of this

Reply:
ok will resolve